.travis.yml

```diff
@@ -38,8 +38,11 @@ before_script:
 script:
   # This is the same as `python setup.py test` with a coverage wrapper.
   - py.test
+  # Test migrations
+  - ./travis_test_migrations.sh
```

Comment:
Any chance we can put this in a `./.travis` directory or name the script `.travis_test_migrations.sh`?

Reply:
Yes that's a good idea, thanks!  I've updated the PR.

Reply:
For what it's worth, I use the `./script/` directory for shell scripts. and then include `ci` somewhere in the script name